docquality.rst

```diff
@@ -91,4 +86,10 @@ installed.  The devguide HTML can be built by running::
 in the checkout directory, which will write the files to the ``_build/html``
 directory.
 
+Changes to the devguide are normally published within a day, on a schedule
+that may be different from the main documentation.
+
+.. _separate repository: https://github.com/python/devguide
+.. _devguide repo: https://github.com/python/devguide
```

Comment:
I couldn't figure out a way to create a reference to an external URL that allows me to use different text, so I duplicated the URL.

AFAIK the only two ways to create a URL are ``` `text <URL>`_ ``` (but with this I have to repeat the URL) or `.. _label: URL` and then ``` `label`_ ``` (but that doesn't let me change the text -- I unsuccessfully tried with ``` `text <label>`_ ```).